roxie/ccd/ccdlistener.cpp

```diff
@@ -142,7 +142,7 @@ class CascadeManager : public CInterface
                     if (!ssock)
                         throw makeStringException(ROXIE_TLS_ERROR, "Roxie CascadeManager failed creating secure socket for roxie control message");
 
-                    int status = ssock->secure_connect();
+                    int status = ssock->secure_connect(2000);
```

Comment:
I do wonder whether the secure socket connect/accept defaults should be different from the jsocket defaults.
These timings (to ssl connect/accept) would generally expect to be shorter than connecting to the raw socket.

And then maybe this/other places, could generally rely on the defaults.